### ISSUE TITLE
a little advice

### ISSUE THREAD

**Issue by ZJsonJun** [zju3dv/pvnet#45]
hi,
I notice that in your code, you read all data containing train&val&test at once for use.
It's a waste of memory, isn't it?
maybe you can improve the code?

**Comment by pengsida** [comment]
We evaluate our network on the validation set during training.
Actually it does not matter much, and you could cancel validation by yourself.
The great amount of training data effectively prevent the overfitting.
